### DISCUSSION TITLE
How to contribute to this project?

### DISCUSSION THREAD

**Discussion by yaten2302** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#217] [Q&A] [answered]
Hi @zerodytrash,

I want to contribute to this project, but I'm a beginner, could you please guide me that from where should I start reading the code and contribute to this project?

**Comment by drunkwinter** [comment, marked as answer]
`src/main.js` is the entry point where everything is imported and used. I wouldn't focus anything related to a refactor, as a rewrite is being worked on currently.
